src/Agent.Worker/Handlers/HandlerFactory.cs

```diff
@@ -117,5 +127,23 @@ public sealed class HandlerFactory : AgentService, IHandlerFactory
             handler.TaskDirectory = taskDirectory;
             return handler;
         }
+
+        /// <summary> 
+        /// This method provides a list of pipeline tasks for which we don't want to display the warning about the Node6 execution handler. 
```

Comment:
Nit: let's also mention that these are in-the-box tasks for visibility

Reply:
Thanks, fixed!